lua/nvim-web-devicons.lua

```diff
@@ -40,12 +40,19 @@ local global_opts = {
   strict = false,
   default = false,
   color_icons = true,
+  variant = "auto",
```

Comment:
Let's not use "auto" please, we don't really need it.

`variant` nil or any other value should be the else case in the if around line 49.

Reply:
Changed to nil, fixed if else logic